index.html

```diff
@@ -98,5 +98,19 @@ <h4>Our Changing Planet</h4>
 
 
     </div>
+
+    <!-- Badge component starts here -->
+    <div class="container flex">
+        <div class="bwbl position-relative">
+            <div class="badge">10+</div>
+            <button>Inbox</button>
+        </div>
+
+        <div class="iwbd position-relative">
+            <div class="badge">12+</div>
```

Comment:
Can't understand iwbt, can you please give it a more meaning full name and instead of using div for badge you can use span